### DISCUSSION TITLE
laravel-translatable support

### DISCUSSION THREAD

**Discussion by clementmas** [filamentphp/filament#373] [Ideas]
Hi,

From what I can see the admin interface is fully translated but does it support managing data in different languages?

Is it already possible to do what [nova-translatable](https://github.com/spatie/nova-translatable) does or is this something planned on the roadmap?

Thanks!

**Comment by danharrin** [comment]
Hey! This is on the roadmap, but needs adding to the page in the docs.

**Reply by Xoshbin** [reply]
Hi, I saw this on the roadmap, I was asking when is it expected to release V2? If it's planned to be released in the next months or not soon, how can I add a button in the admin panel to change the locales after I installing the spatie/laravel-translatable package by myself please?
Thank you
